commands/default_cmdsets.py

```diff
@@ -38,6 +39,7 @@ def at_cmdset_creation(self):
         self.add(chartraits.CharTraitCmdSet())
         self.add(room_exit.AinneveRoomExitsCmdSet())
         self.add(building.AinneveBuildingCmdSet())
+	self.add(skills.SkillCmdSet())
```

Comment:
This looks like it could be spacing issue that would cause errors. Please double-check it.